solidity/orders/src/Create2.sol

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity >=0.8.25 <0.9.0;
+
+contract Create2 {
```

Comment:
Can you pls keep here the link to the source as comment?
`// From https://book.getfoundry.sh/tutorials/create2-tutorial`